nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/test/java/org/nuxeo/ecm/platform/tag/AbstractTestTagService.java

```diff
@@ -101,6 +102,12 @@
     @Inject
     protected NXAuditEventsService auditEventsService;
 
+    /**
+     * @since 11.1
+     */
+    @Inject
+    protected LogCaptureFeature.Result logCaptureResult;
```

Comment:
How is that part of the cleanup?
Shouldn't it be embedded in the 2nd commit? See https://github.com/nuxeo/nuxeo/pull/3601/commits/7b4b9eadceeeb696809b719026e9e81be7ef6f6b#diff-7a3e8bc7594906000cfc1478ab972578R926

Reply:
OK, I get it, you first want to move the injected `logCaptureResult` tot the abstract class, but IMHO, it should all be part of the same commit:
- It isn't format nor cleanup.
- Commits are not independent, if you take the second one without the first one it won't compile I guess.

In fact, you also need to move `@Features(LogCaptureFeature.class)` form `TestFacetedTagService` to `AbstractTestTagService`. In the current state, I'm pretty sure that trying to use `logCaptureResult` in another subclass than `TestFacetedTagService` would fail.

Reply:
Copy that !